canonical-web-and-design/jaas-dashboard
# Add new sharing user

## Done

Add new sharing user to model details

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Open a model
- Click the 'Share model' button in the InfoPanel
- Check that you can add/remove users

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1744,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1742,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1738.
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1740,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1745
